packages/vuetify/src/util/helpers.ts

```diff
@@ -447,3 +447,7 @@ export function mergeDeep (
 
   return source
 }
+
+export function fillArray<T> (length: number, obj: T) {
+  return Array(length).fill(obj)
```

Comment:
This requires a polyfill as it's not compatible with ie11 by default. Is this something we want to do? If so, we should add an alert to the doc pages. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill#Polyfill

Reply:
We already have one other instance of `Array.fill` in the code. And it looks like at least IE 11.836.18362.0 _does_ support it when checking playground.